projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-list/query-list.component.html

```diff
@@ -0,0 +1,29 @@
+<ng-container *ngIf="queryListData">
+  <table class="govuk-table query-list__table">
+    <caption class="govuk-table__cell query-list__caption query-list__row">
+      <div>{{ queryListData.partyName }}</div>
+    </caption>
+    <thead class="govuk-table__head">
+    <tr class="govuk-table__row query-list__row">
+      <th scope="col" class="govuk-table__header">Queries</th>
```

Comment:
@MateiHMCTS - Lines 8 to 12 need to use the Welsh translation pipe, so it should look something like this: `{{'Queries' | rpxTranslate}}`